src/stratis_cli/_actions/_stratisd_version.py

```diff
@@ -0,0 +1,23 @@
+"""
+Check version of stratisd
+"""
+
+
+def check_stratisd_version():
+    """
+    Checks that the version of stratisd that is running is compatible with
+    this version of the CLI
+
+    :raises StratisCliStratisdVersionError
+    """
+    from ._connection import get_object
+    from ._constants import TOP_OBJECT
+    from ._data import MAXIMUM_STRATISD_VERSION, MINIMUM_STRATISD_VERSION, Manager
+    from .._errors import StratisCliStratisdVersionError
```

Comment:
Only the `._data` import needs to be loaded inside the method. The rest can be top level imports.